sources/packages/backend/apps/db-migrations/src/sql/Applications/Add-application-versioning-columns.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE
+  sims.applications
+ADD
+  COLUMN parent_application_id INT REFERENCES sims.applications(id);
```

Comment:
As a lessons learnt from offering, it would be good to enforce the new columns not NULL after the UPDATE SQL so that it ensures the following scenarios have these fields populated(it does not get missed).

1. Create Draft Application(Starting a new application).
2. Submit application from Draft Status.
3. Re-submit application.